src/Yoti/ActivityDetails.php

```diff
@@ -138,6 +144,36 @@ public static function constructFromAttributeList(AttributeList $attributeList,
         return $inst;
     }
 
+    public function createApplicationProfile($attributeList)
+    {
+        if (!($attributeList instanceof AttributeList)) {
+            return;
+        }
+        $profileAttributes = [];
+        $anchorProcessor = new AnchorProcessor();
+
+        foreach($attributeList->getAttributes() as $attr) { /** @var ProtobufAttribute $attr */
+            $attrName = $attr->getName();
```

Comment:
I agree, we could move the creation of Yoti attribute into a method, but it doesn't have to be inside an AttributeConverter as we are not really converting directly ProtobufAttribute, it's going through some processes.
If we want to have it outside ActivityDetails so it can be reused else where, then I could agree.